utils/python/CIME/hist_utils.py

```diff
@@ -105,15 +90,25 @@ def _hists_match(model, hists1, hists2, suffix1="", suffix2=""):
     >>> hists2 = ['cpl.h2.nc.SUF2', 'cpl.h3.nc.SUF2', 'cpl.h4.nc.SUF2']
     >>> _hists_match('cpl', hists1, hists2, 'SUF1', 'SUF2')
     (['FOO.G.cpl.h1.nc.SUF1'], ['cpl.h4.nc.SUF2'], [('FOO.G.cpl.h2.nc.SUF1', 'cpl.h2.nc.SUF2'), ('FOO.G.cpl.h3.nc.SUF1', 'cpl.h3.nc.SUF2')])
+    >>> hists1 = ['cam.h0.1850-01-08-00000.nc']
```

Comment:
Now if hist1 has single instance and hist2 has multiinstance or visa-versa the single will be compared to all of the multis.  If both have multiinstance each instance is compared